Source/src/core/rendering/Skybox.h

```diff
@@ -17,11 +17,19 @@ namespace Hachiko
         void ChangeCubeMapSide(UID texture_uid, TextureCube::Side cube_side);
         void ReleaseCubemap();
 
+        [[nodiscard]] bool IsIBLActive() const
+        {
+            return activate_ibl;
+        }
         const TextureCube& GetCube() const
         {
             return cube;
         }
 
+        void ActivateIBL() 
```

Comment:
Just out of curiosity, how do we handle Deactivation of IBL?

Reply:
The deactivation is handled by UI as the activation. But it is going to be a good idea to add a parameter there to have more control.